vscode-dotty/src/extension.ts

```diff
@@ -17,12 +17,9 @@ export function activate(context: ExtensionContext) {
   outputChannel = vscode.window.createOutputChannel('Dotty Language Client');
 
   const artifactFile = `${vscode.workspace.rootPath}/.dotty-ide-artifact`
+  const defaultArtifact = "ch.epfl.lamp:dotty-language-server_0.8:0.8.0-bin-SNAPSHOT"
```

Comment:
If this get in, we'd need to find a way to keep these values up-to-date with the reality. When's the vscode extension published? Every 6 weeks, too?

Reply:
You can get the current nightly base of dotty from http://dotty.epfl.ch/versions/latest-nightly-base, we could also have a file to get the current release version.